pkg/openthread/contrib/platform_radio.c

```diff
@@ -172,20 +182,35 @@ void recv_pkt(otInstance *aInstance, netdev_t *dev)
     otPlatRadioReceiveDone(aInstance, res > 0 ? &sReceiveFrame : NULL, res > 0 ? OT_ERROR_NONE : OT_ERROR_ABORT);
 }
 
+/* create a fake ACK frame: currently does not directly provide a received ACK */
+static inline void _create_fake_ack_frame(bool ackPending)
+{
+    sAckFrame.mPsdu[0] = IEEE802154_FCF_TYPE_ACK;
+    if (ackPending) {
+        sAckFrame.mPsdu[0] |= IEEE802154_FCF_FRAME_PEND;
+    }
+    sAckFrame.mPsdu[1] = 0;
+    sAckFrame.mPsdu[2] = sTransmitFrame.mPsdu[IEEE802154_DSN_OFFSET];
+
+    sAckFrame.mInfo.mRxInfo.mRssi = OT_RADIO_RSSI_INVALID;
+}
+
 /* Called upon TX event */
-void send_pkt(otInstance *aInstance, netdev_t *dev, netdev_event_t event)
+void sent_pkt(otInstance *aInstance, netdev_t *dev, netdev_event_t event)
```

Comment:
~~Why this change in name?~~
Please see comment in declaration.

Reply:
Checked